packages/common/src/pipes/number_pipe.ts

```diff
@@ -101,6 +101,22 @@ export class DecimalPipe implements PipeTransform {
     locale = locale || this._locale;
 
     try {
+      if (typeof value === 'string' && value.includes('.')) {
+        const parts = value.split('.');
+        if (parts.length != 2) {
+          throw new Error(`${value} is not a number`);
+        }
+        const integerPart = parts[0];
+        const fractionPart = '0.' + parts[1];
+
+        const integerStr = formatNumber(strToNumber(integerPart), locale, digitsInfo).split('.')[0];
+        const fractionStr = formatNumber(strToNumber(fractionPart), locale, digitsInfo).split('.')[1];
+
+        if (typeof fractionStr === 'undefined') {
```

Comment:
Looks like this can be simplified. 
```suggestion
        if (fractionStr === undefined) {
```